python/qpid_dispatch/management/client.py

```diff
@@ -76,6 +76,16 @@ def delete(self):
 class Node(object):
     """Client proxy for an AMQP management node"""
 
+    def clean_attrs(self, attrs):
+        BOOL_VALUES = {"yes": 1, "true": 1, "on": 1, "no": 0, "false": 0,
+                       "off": 0}
+        if isinstance(attrs, dict):
+            for key in attrs.keys():
+                if attrs[key] in BOOL_VALUES.keys():
+                    attrs[key] = bool(BOOL_VALUES[attrs[key]])
```

Comment:
Instead of putting 1 or 0 as the values for BOOL_VALUES, why not use the python bools True and False?  Then you don't need to cast to bool()